vulture/core.py

```diff
@@ -90,12 +89,16 @@ def read_file(filename):
 
 
 def _get_suitable_whitelists(defined_imports):
-    all_whitelists = set(pkg_resources.resource_listdir('vulture',
-                                                        'whitelists'))
-    defined_imports = set([imp + '.py' for imp in defined_imports])
-    common = sorted(all_whitelists.intersection(defined_imports))
-    whitelist_paths = [os.path.join('whitelists', name) for name in common]
-    return whitelist_paths
+    wh_names = []
+    defined_imports = ['wh_' + imp for imp in defined_imports]
+    wh_mods = __import__('whitelists', globals(), locals(), defined_imports, 1)
```

Comment:
> It would be good if we could avoid prepending all whitelists with a prefix like "wh_".

Okay. I did this to prevent confusions while debugging.
> Can we import the modules one by one using something like __import__('vulture.whitelists.whitelist_name', ...)?  If we prepend "vulture.whitelists", maybe we don't have to rename the files.

Is this required only so that we don't have to rename the files?
If yes, then we can just remove the line:
```python
defined_imports = ['wh_' + imp for imp in defined_imports]
```
and then we can rename the files back to 'sys.py', etc.

Let me make changes in the PR itself, so we can have better insights.